src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java

```diff
@@ -12,5 +12,5 @@
 public class Frequency {
     private Unit unit;
     private int value;
-    public static Frequency ZERO_HOUR = new Frequency(Unit.HOUR, 0);
+    public static Frequency ONE_HOUR = new Frequency(Unit.HOUR, 1);
```

Comment:
best not define as constants (unless meant as default value).  in that case, "Unit.HOUR" can be read from application properties.

Reply:
These are default values for now, hence defined as above.